### DISCUSSION TITLE
Shows the disk SMART wearout percentage on a node's Disk

### DISCUSSION THREAD

**Discussion by dougiteixeira** [dougiteixeira/proxmoxve#272] [Feature request] [answered]
@dougiteixeira Proxmox itself shows the disk SMART wearout percentage on a node's Disk area, usually as the last column. Can this be exposed as a sensor value?

BTW, great add-on!

_Originally posted by @scottcopus in https://github.com/dougiteixeira/proxmoxve/discussions/176#discussioncomment-8574646_

**Comment by dougiteixeira** [comment]
@scottcopus Can you download the diagnostics and send it along with a screenshot of the information?

**Reply by scottcopus** [reply]
Please explain what diagnostics? Thanks.

**Reply by dougiteixeira** [reply]
See: https://www.home-assistant.io/docs/configuration/troubleshooting/#download-diagnostics

**Comment by dougiteixeira** [comment, marked as answer]
Added in #305. It will be available in the next version (3.4.2).

**Reply by AF360** [reply]
I just installed the latest version of the integration (3.4.4) but I can't see any SMART-entities or disk wearout-information. Neither enabled nor disabled entities.

Proxmox does show the SMART information (under Disks -> Show S.M.A.R.T values) and also shows them with smartctl in a shell.

@dougiteixeira will this still be added later or not at all or what am I missing?

Of course I can add a script to push that information to HA like I do with host CPU temperature but that's not required if it's included in the integration.

**Reply by dougiteixeira** [reply]
This has already been implemented. Your HD information probably does not follow the standard.

If you wish, generate a diagnostic file and open an issue for further evaluation.

**Reply by AF360** [reply]
I had not turned on "Enable physical disk information" in the integration. 
As soon as I did I was able to see the disk SMART status information and it's working perfectly.
Sorry for the noise.
